docs/DEV_README.md

```diff
@@ -33,6 +33,13 @@ See [**Frontitude CLI**](./FRONTITUDE_CLI.md) to know how to update your `string
 - [__Remote reference__](./REMOTE_PUBLISHING.md)
 - [__Workflow reference__](./ACTION_PUBLISHING.md)
 
+## Spotless integration
+
+[Spotless](https://github.com/diffplug/spotless) plugin for Gradle.
+
+Run ./gradlew spotlessCheck to detect issues or ./gradlew spotlessApply to apply fixing for the issues detected automatically.
```

Comment:
you can use this quotes \`  \`  wrapping the text you want to display as command, like `./gradlew spotlessCheck` and  `./gradlew spotlessApply`, so it is easier to see the commands

Reply:
True ! Forgot to highlight the commands ! Thanks !